test/verify/check-services

```diff
@@ -470,7 +470,7 @@ WantedBy=default.target
         b.click("#timer-save-button")
         b.wait_popdown("timer-dialog")
         b.wait_present(svc_sel('boot_timer.timer'))
-        m.execute("systemctl enable boot_timer")
+        m.execute("systemctl enable boot_timer.timer")
```

Comment:
Yeah, I agree. It would seem strange to me to create a timer without the intention of actually using it.